### DISCUSSION TITLE
Ability to use within sub-folders of a vault

### DISCUSSION THREAD

**Discussion by micahwalter** [Vinzent03/obsidian-git#431] [General]
I am curious if it would be useful to others to use this plugin for specific sub-folders within a vault. For example, I have a large vault of notes and I like to pull in other repositories as sub-folders to work on them within Obsidian. I use Git to track these repos elsewhere, but it would be nice if this plugin could manage the repo within Obsidian. 

Could this work?

**Comment by Vinzent03** [comment]
That's already possible via the custom base path setting.

**Reply by micahwalter** [reply]
Right, I saw that, but it's only available for a single repo. My though was to have multiple repos within a single vault.

**Reply by Vinzent03** [reply]
Ah okay, that's not currently planned.

**Comment by ArneNostitz** [comment]
can i have a Sub-Vault synching to a different repo?

Usecase: 
a main vault synched to "main-Repo",
a subfolder of the main vault opened as a vault - synched to a "project-repo" (in this sub-repo are only project related files)

this way i could work and reference on the main vault and share files with coworkers without them having access to all my files

or would that lead to massive problems?

**Reply by Vinzent03** [reply]
I think Git submodules could solve your issue.

**Reply by ArneNostitz** [reply]
i will look into that - did not understand it yet :) thanks

**Reply by AmauryVanEspen** [reply]
Hi @Vinzent03 
I would be curious to know more about subfolders sync with gitsubmodule or subrepo.
